src/main/java/org/jboss/xavier/integrations/route/MainRouteBuilder.java

```diff
@@ -91,8 +92,9 @@ public void configure() {
                 .to("log:INFO?showBody=true&showHeaders=true")
                 .end();
 
-        from("kafka:" + kafkaHost + "?topic={{insights.kafka.upload.topic}}&brokers=" + kafkaHost + "&autoOffsetReset=latest&autoCommitEnable=true")
+        from("kafka:" + kafkaHost + "?topic={{insights.kafka.upload.topic}}&brokers=" + kafkaHost + "&autoOffsetReset=earliest&autoCommitEnable=true")
```

Comment:
Sure, it will take time though.
You mean having a test comparing the 2 approaches, because we didnt know if the previous one was doing what we wanted....to do that we need several messages, and a retarded connection by the kafka listener in order to see which one is consuming more messages